src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  box-sizing: border-box;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  width: 200px;
+  padding: 32px 16px 0;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-top: 40px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+   }
+
+  &__code {
+    margin-top: 4px;
+    color: $secondary;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
```

Comment:
It's a default value, remove it
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#formal_definition
```suggestion
```

Reply:
fix in all cases